recipes/ansys-rocky-core/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "ansys-rocky-core" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/ansys_rocky_core-{{ version }}.tar.gz
+  sha256: 7ba412d28fcebc627ed8157aa9e9128573f36e9dd04021cc270b602574ffa409
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >={{ python_min }},<4.0
```

Comment:
Looks like `python_min` is not defined when linting and is empty, causing an error. BTW, you don't need the `,4`. It is very unlikely that things in this package will break and/or live that long to see breakages.